src/overlays/actors/ovl_En_Daiku/z_en_daiku.h

```diff
@@ -5,9 +5,44 @@
 
 struct EnDaiku;
 
+typedef void (*EnDaikuActionFunc)(struct EnDaiku*, GlobalContext*);
+
+#define ENDAIKU_FF(thisx) ((thisx)->params & 0xFF);
+#define ENDAIKU_FF00(thisx) (((thisx)->params >> 8) & 0xFF);
```

Comment:
```suggestion
#define ENDAIKU_GET_PARAMS_FF(thisx) ((thisx)->params & 0xFF);
#define ENDAIKU_GET_PARAMS_FF00(thisx) (((thisx)->params >> 8) & 0xFF);
```
I wonder if something like this would be clearer. There should be an indication in the name that this macro is for params. Kenix or others might have a better idea/more standard name we want to start adopting. Possibly a discussion for discord

Reply:
Undocumented params access should follow the format `OVLNAME_GET_BITS`